### ISSUE TITLE
Cant make this module work

### ISSUE THREAD

**Issue by damdafayton** [yifaneye/react-gallery-carousel#73]
Hi,
the live example looks really nice but 
- the example in clone from repo doesn't work with `npm start`
- it doesn't render correctly when I install the library into my own project.

I don't know whats wrong but its pity that such nice looking library can not be implemented smoothly.

**Comment by yifaneye** [comment]
Hi @damdafayton ! Thank you for raising this issue up.

I can't reproduce your bugs. Can you give more information?

**Comment by yifaneye** [comment]
Hi @damdafayton !

Regarding
> the example in clone from repo doesn't work with npm start
It should be fixed in Issue #72 

Regarding
> it doesn't render correctly when I install the library into my own project.
Would you like to give me some more information? I am happy to help you. 😃

**Comment by hamez0r** [comment]
~~Hi! I'm trying to make it work here, but no luck :(
https://codesandbox.io/s/rough-silence-ggzckp?file=/src/App.js~~

I wasn't importing the CSS. @damdafayton maybe that was your issue too?

**Comment by yifaneye** [comment]
Yes. Currently, we need to import the CSS from the package by adding `import 'react-gallery-carousel/dist/index.css';`. See https://github.com/yifaneye/react-gallery-carousel#usage
Adding the line at https://codesandbox.io/s/runtime-rgb-pk9mqj?file=/src/App.js makes the carousel work.
